api/tests/src/services/opportunities_v1/test_opportunity_version.py

```diff
@@ -25,12 +25,13 @@ def test_save_opportunity_version(db_session, enable_factory_create):
     expected = {
         "agency": agency.agency_code,
         "summary": None,
-        "category": OpportunityCategory.MANDATORY,
+        "category": str(OpportunityCategory.MANDATORY),
```

Comment:
This is already a string / shouldn't be hitting any issues with comparison. If it is, that's 
```suggestion
        "category":OpportunityCategory.MANDATORY,
```
If you saw a weird comparison / it looked like it wouldn't match when you were fixing the UUIDs, that's normal. Because they're `StrEnum` it'll be like "dict1 doesn't match dict2" and print the two dicts, one will be a string, and the other is like `<MyEnum.SOMETHING: 'something'>` but it's really also a string, that's just the repr function.